src/libstd/primitive_docs.rs

```diff
@@ -769,19 +769,57 @@ mod prim_str {}
 mod prim_tuple {}
 
 #[doc(primitive = "f32")]
-/// The 32-bit floating point type.
+/// A 32-bit floating point type (specifically, the "binary32" type defined in IEEE 754-2008).
+///
+/// This type can represent a wide range of decimal numbers, like `3.5`, `27`,
+/// `-113.75`, `0.0078125`, `34359738368`, `0`, `-1`. So unlike integer types
+/// (like `i32`), floating point types can represent non-integer numbers, too.
```

Comment:
unlike followed by *(like ...)* made me stop reading the sentence for a moment to ensure that I am parsing it correctly, i.e. whether it meant *but like ...*. perhaps it could be replaced with *(such as ...)* or *(e.g. ...)* or removed altogether.

Reply:
Good idea, thanks for the feedback! I pushed the "such as" suggestion.

Reply:
Odd, I'm not seeing that in the merge commit.

Reply:
Oh damn, I think I screwed up a force push. None of the review-fixes are included :( 
Will open a follow up PR to fix all of them.

Reply:
See #75400